inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/sidebar/LearningCurveChartPanel.html

```diff
@@ -18,15 +18,15 @@
 -->
 <html xmlns:wicket="http://wicket.apache.org">
 <wicket:panel>
-  <div class="panel panel-default panel-small">
-    <div class="panel-heading">
-      <h3 class="panel-title">
-        <wicket:message key="learningCurve" />
-      </h3>
-    </div>
-    <div class="panel-body">
-      <div wicket:id="chart-container"></div>
-    </div>
-  </div>
+	<div class="panel panel-default panel-small">
```

Comment:
This file contains lots of whitespace changes that need to be reverted. Please note that HTML files are indented using spaces only and using 2 spaces for an indentation.